api/src/main/java/org/openmrs/module/metadatamapping/api/db/hibernate/HibernateMetadataMappingDAO.java

```diff
@@ -195,11 +195,7 @@ public MetadataTermMapping getMetadataTermMapping(MetadataSource metadataSource,
 				        + metadataTermMapping.getUuid() + " refers to type " + metadataTermMapping.getMetadataClass());
 			}
 			metadataItem = internalGetByUuid(type, metadataTermMapping.getMetadataUuid());
-			if (metadataItem != null && metadataItem.isRetired()) {
-				metadataItem = null;
-			}
```

Comment:
Have you run the unit tests? It seems this change would break this test: https://github.com/openmrs/openmrs-module-metadatamapping/blob/master/api/src/test/java/org/openmrs/module/metadatamapping/api/MetadataMappingServiceTest.java#L791

Please also make sure to run all the other tests.